tests/integration/ceph_mgr_test.go

```diff
@@ -122,19 +119,61 @@ func (s *CephMgrSuite) execute(command []string) (error, string) {
 	return s.installer.Execute("ceph", orchCommand, s.namespace)
 }
 
+func (s *CephMgrSuite) enableRookOrchestrator() error {
+	logger.Info("Enabling Rook orchestrator module: <ceph mgr module enable rook --force>")
+	err, output := s.installer.Execute("ceph", []string{"mgr", "module", "enable", "rook", "--force"}, s.namespace)
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to enable rook orchestrator module: %q", err)
+		return err
+	}
+	logger.Info("Setting orchestrator backend to Rook .... <ceph orch set backend rook>")
```

Comment:
The test log should already show the ceph command executed, I don't think you need this

Reply:
I can see the logs of the different Ceph pods logs, events, and pod descriptions in "tests/integration/_output/tests". But I not able to see any log for the test itself. So for me is useful to have this information in the standard output. 
Where can i find the log file for the test??

Reply:
It should just be in stdout like this output. Let's chat more about it though.